products/compute/api.yaml

```diff
@@ -3711,31 +3711,20 @@ objects:
       - !ruby/object:Api::Type::Array
         name: 'guestOsFeatures'
         description: |
-            A list of features to enable on the guest OS. Applicable for
-            bootable images only. Currently, only one feature can be enabled,
-            VIRTIO_SCSI_MULTIQUEUE, which allows each virtual CPU to have its
-            own queue. For Windows images, you can only enable
-            VIRTIO_SCSI_MULTIQUEUE on images with driver version 1.2.0.1621 or
-            higher. Linux images with kernel versions 3.17 and higher will
-            support VIRTIO_SCSI_MULTIQUEUE.
-
-            For new Windows images, the server might also populate this field
-            with the value WINDOWS, to indicate that this is a Windows image.
-            This value is purely informational and does not enable or disable
-            any features.
+            A list of features to enable on the guest operating system.
+            Applicable only for bootable images.
         item_type: !ruby/object:Api::Type::NestedObject
           properties:
             - !ruby/object:Api::Type::Enum
               name: 'type'
               description: |
-                The type of supported feature. Currently only
-                VIRTIO_SCSI_MULTIQUEUE is supported. For newer Windows images,
-                the server might also populate this property with the value
-                WINDOWS to indicate that this is a Windows image. This value is
-                purely informational and does not enable or disable any
-                features.
+                The type of supported feature.
```

Comment:
Do you mind listing the feature values or linking to a reference of which values are available in the description?